apps/web/src/pages/Pools/EditPoolPage/components/AboutUsSection/Display.tsx

```diff
@@ -3,14 +3,18 @@ import { useIntl } from "react-intl";
 import { commonMessages } from "@gc-digital-talent/i18n";
 import { Well } from "@gc-digital-talent/ui";
 import { RichTextRenderer, htmlToRichTextJSON } from "@gc-digital-talent/forms";
+import { EditPoolAboutUsFragment } from "@gc-digital-talent/graphql";
 
 import ToggleForm from "~/components/ToggleForm/ToggleForm";
 import processMessages from "~/messages/processMessages";
 import { hasAllEmptyFields } from "~/validators/process/aboutUs";
 
 import { DisplayProps } from "../../types";
 
-const Display = ({ pool, subtitle }: DisplayProps) => {
+const Display = ({
+  pool,
+  subtitle,
+}: { pool: EditPoolAboutUsFragment } & Pick<DisplayProps, "subtitle">) => {
```

Comment:
I think this could also be a good spot for a generic.

```ts
type DisplayProps<P> = {
  subtitle?: ReactNode;
  pool: P;
}
```

Reply:
Likewise https://github.com/GCTC-NTGC/gc-digital-talent/pull/11015/commits/67bcfad8899e6c4468abea203442d07030b13ec4